cpp/benchmarks/join/generate_input_tables.cuh

```diff
@@ -22,13 +22,12 @@
 
 #include <rmm/exec_policy.hpp>
 
+#include <curand.h>
+#include <curand_kernel.h>
 #include <thrust/distance.h>
 #include <thrust/iterator/counting_iterator.h>
 #include <thrust/sequence.h>
```

Comment:
Should CCCL libraries be its own category too? Since we use them pervasively, placing them in a separate group help identifying if there are unused headers in the future.

Reply:
My vote would be:
- Thrust/CUB in its own category
- Other "dot" headers from the system
- libcudacxx `<cuda/...>`
- STL (no dot)

Reply:
I would also accept:
- CCCL (Thrust/CUB/libcudacxx) in its own category
- Other "dot" headers from the system
- STL (no dot)

Reply:
I also vote for `cuda/, thrust/, cub/` in a single category

Reply:
Done. I like this too. I also put cuco and CUDA headers (cuda_runtime.h etc.) in the same group. This adds one more regex (11), but same number of groups (10).